src/components/policy/test/policy_manager_impl_test.cc

```diff
@@ -37,28 +37,29 @@
 
 #include "json/reader.h"
 #include "gtest/gtest.h"
+#include "gmock/gmock.h"
+#include "mock_policy_listener.h"
+#include "mock_pt_representation.h"
+#include "policy/test/include/mock_cache_manager.h"
```

Comment:
@OHerasym Please edit header path . Shoudl be used the following template : `<directory/file.h>`. Path to `directory` should be specified in CMakeList